cmd/snap-exec/main.go

```diff
@@ -32,7 +32,6 @@ import (
 	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/snap"
-	"github.com/snapcore/snapd/snap/snapdir"
```

Comment:
is this going to be imported some other way? otherwise the hook will not be set

Reply:
Hrm, I could put a `snap.NewContainerFromDir = snapdir.NewContainerFromDir` in the `init` in `cmd/snap-exec/main.go`, is that how we'd normally do this?

Reply:
Done in 8d483823c02711d8d89b9d3979cf7131b1e14d29, lmk if we should do this a better way.

Reply:
see for example:

./cmd/snap/cmd_sign_build.go:	_ "golang.org/x/crypto/sha3"